settings/Developer.setting.php

```diff
@@ -79,6 +79,19 @@
     'is_contact' => 0,
     'description' => "Set this value to Yes if you want to display a backtrace listing when a fatal error is encountered. This feature should NOT be enabled for production sites",
   ),
+  'civicrmEnvironment' => array(
+    'group_name' => 'Developer Preferences',
+    'group' => 'developer',
+    'name' => 'civicrmEnvironment',
```

Comment:
Name shouldn't have civicrm preference as that is non-standard. In order to make sense of Boolean value, let's follow field naming pattern from schema and use isProductionEnvironment.